lib/data/people.js

```diff
@@ -137,6 +138,17 @@ export const team = [
       { name: "Site", url: "https://yob.id.au" }
     ],
     colour: "rgb(34,139,34)"
+  },
+  {
+    photo: photoFred,
+    name: "Fred Showell",
+    bio: "Hi, I’m Fred. I enjoy designing great experiences and finding ways to allow for moments of delight within a software product. I live in Hobart with my wife, two kids, and an unhealthy quantity of weird old LPs.",
```

Comment:
Looks like the LPs is sad town on the next line.

```suggestion
    bio: "Hi, I’m Fred. I enjoy designing great experiences and finding ways to allow for moments of delight within a software product. I live in Hobart with my wife, two kids, and an unhealthy quantity of weird old LPs.",
```